rfcs/0007-consistent-vehicle-ids.md

```diff
@@ -0,0 +1,185 @@
+for modes to be served by multiple vendors (such as Yankee for shuttle
+buses). Although this flexibility may not be needed for the other modes at
+present, we have changed Commuter Rail vendors in the past and so it is not
+outside the realm of possiblity.
+
+# Prior art
+[prior-art]: #prior-art
+
+<!--
+Discuss prior art, both the good and the bad, in relation to this proposal. A few examples of what
+this can include are:
+
+- Can we learn something about this proposal from other projects in CTD?
+- Can we learn something about this proposal from other transit agencies?
+- Can we learn something about this proposal from past experiences in other jobs or projects?
+
+This section is intended to encourage you as an author to think about the lessons from other places,
+and provide readers of your RFC with a fuller picture.
+
+If there is no prior art, that is fine.
+
+Note that while precedent is some motivation, it does not on its own motivate an RFC. -->
+
+No prior art that I am aware of.
+
+# Unresolved questions
+[unresolved-questions]: #unresolved-questions
+
+<!--
+- What parts of the design do you expect to resolve through the RFC process before this gets merged?
+- What parts of the design do you expect to resolve through the implementation of this feature
+  before stabilization?
+- What related issues do you consider out of scope for this RFC that could be addressed in the
+  future independently of the solution that comes out of this RFC? -->
+
+- when will teams find the time in their sprints to make this change?
+- will a developer external to the team (such as Paul) draft pull requests?
+
+# Future possibilities
+[future-possibilities]: #future-possibilities
+
+<!--
+Think about what the natural extension and evolution of your proposal would be and how it would
+affect the project as a whole in a holistic way. Try to use this section as a tool to more fully
+consider all possible interactions with the project in your proposal. Also consider how this all
+fits into the roadmap for the project.
+
+This is also a good place to "dump ideas", if they are out of scope for the RFC you are writing but
+otherwise related.
+
+If you have tried and cannot think of any future possibilities, you may simply state that you cannot
+think of anything.
+
+Note that having something written down in the future-possibilities section is not a reason to
+accept the current or a future RFC; such notes should be in the section on motivation or rationale
+in this or subsequent RFCs. The section merely provides additional information. -->
+
+We could consider doing something similar for trip IDs in the future. At the
+moment, MBTA and Keolis are both using HASTUS for scheduling, which uses
+integer trip IDs. They do not currently overlap, but we don't know whether
+that's a possiblity in the future. Changing the trip IDs would be a much
+larger lift however, as not all applications (TransitMaster) support
+non-integer trip IDs.
```

Comment:
Aren't TransitMaster trip IDs distinct from HASTUS/GTFS trip IDs anyway?

Reply:
Yes, but we're in the process of asking TransitMaster to import our GTFS file and one of the sticking points is that we have non-integer trip IDs that they can't import.